tools/update_vcpkg.py

```diff
@@ -0,0 +1,168 @@
+import re
```

Comment:
the update_vcpkg.py file is not executable can you add that? 
```suggestion
#!/usr/bin/env python3
import re
```